endpoints/openrtb2/video_auction.go

```diff
@@ -471,14 +471,7 @@ func mergeData(videoRequest *openrtb_ext.BidRequestVideo, bidRequest *openrtb.Bi
 		bidRequest.Device = &videoRequest.Device
 	}
 
-	if &videoRequest.User != nil {
-		bidRequest.User = &openrtb.User{
-			BuyerUID: videoRequest.User.Buyeruids["appnexus"], //TODO: map to string merging
-			Yob:      videoRequest.User.Yob,
-			Gender:   videoRequest.User.Gender,
-			Keywords: videoRequest.User.Keywords,
-		}
-	}
+	bidRequest.User = videoRequest.User
```

Comment:
@hhhjort Is a direct assignment here good with you or do you recommend a shallow copy?

Reply:
That should be fine. Nothing should be touching the `User` through the `videoRequest` pointer.